pints/_log_likelihoods.py

```diff
@@ -105,6 +132,11 @@ class ARMA11LogLikelihood(pints.ProblemLogLikelihood):
         A :class:`SingleOutputProblem` or :class:`MultiOutputProblem`. For a
         single-output problem three parameters are added (rho, phi, sigma),
         for a multi-output problem 3 * ``n_outputs`` parameters are added.
+
+    References
+    ----------
+    .. [2] Hamilton, James D. Time series analysis. Vol. 2. New Jersey:
```

Comment:
I'm not 100% sure but I think this can be [1] too? So numbering per class rather than for the whole module? Correct me if I'm wrong please!

Reply:
I also thought calling them both [1] would be better, especially as it is the same book. But Sphinx does throw a warning if you do that and messes up the HTML. 

``pints/_log_likelihoods.py:docstring of pints.ARMA11LogLikelihood:15: WARNING: Duplicate target name, cannot be used as a unique reference: "1".``